test/Integration/IntegrationTestBase.cs

```diff
@@ -265,6 +265,20 @@ private static string GetFunctionsCoreToolsPath()
                     }
                     throw new FileNotFoundException($"Azure Function Core Tools not found at {funcPath} or {programFilesFuncPath}");
                 }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
Does OSPlatform only has OSX for mac? Just making sure that it's inclusive for macos11/12/etc

Reply:
There seems to be `FreeBSD, Linux, OSX, Windows, and Create OS` as options for OSPlatform. 

I believe that OSX should be inclusive of any version of Mac.